static/app/components/panels/panelAlert.tsx

```diff
@@ -18,6 +18,7 @@ const DEFAULT_ICONS = {
 const PanelAlert = styled(({icon, ...props}: Props) => (
   <Alert {...props} icon={icon || DEFAULT_ICONS[props.type!]} system />
 ))`
+  align-items: center;
```

Comment:
minor bug fix

Reply:
I'm pretty sure we intentionally do not center this, cc @doralchan

Reply:
We actually explicitly align the icon to the first line here https://github.com/getsentry/sentry/blob/c559d60da7541ea9d329d7fc52d4bbc31d37fb38/static/app/components/alert.tsx#L29-L32

Reply:
are you sure? it does not look nice
![image](https://user-images.githubusercontent.com/29228205/122372172-c0991500-cf60-11eb-89cb-ea6603246c22.png)

Reply:
Yeah I am sure. You're right though in the case where there is a button which makes the text bigger, it is indeed a problem.

I think we need a different solution, because I've seen us change the alignment of these back and forth a few times. One person centers it, another person uncenters it.

Reply:
ok I think it's better to remove this change from this PR now and discuss it with the design team

Reply:
https://github.com/getsentry/sentry/pull/17840 Here's where we centered things.

I would suggest maybe a prop named `iconLineHeight` so for cases like that button we can adjust that

Reply:
sure.. I will address it in a follow-up PR 😉